finality-aleph/src/crypto.rs

```diff
@@ -24,6 +24,13 @@ impl From<AuthoritySignature> for Signature {
     }
 }
 
+// This is here just for a compatibility hack, remove when removing legacy/v1 authentications.
```

Comment:
Maybe mark it as TODO, coz otherwise I think we will forget about it

Reply:
It's already mentioned explicitly in A0-1718. :3